src/config.rs

```diff
@@ -49,5 +49,22 @@ pub fn get_backups_flder() -> PathBuf {
 
 #[cfg(target_family = "unix")]
 pub fn get_boilr_links_path() -> PathBuf {
-    get_config_folder().join("links").to_path_buf()
+    // Remove redundant_clone. Refer: https://rust-lang.github.io/rust-clippy/master/index.html#redundant_clone
+    get_config_folder().join("links")
+}
+
+
+#[cfg(test)]
+mod tests{
+    use std::path::PathBuf;
+
+    use super::get_config_folder;
+
+    #[test]
+    fn check_return_config_format(){
+        let path=get_config_folder();
+        let val=std::env::var("HOME").unwrap();
```

Comment:
This will always fail on Windows, since there is no HOME variable.

Reply:
I'd like to change this test case just cover the Unix environment.